estate/views/estate_menus.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<odoo>
+    <menuitem id="estate_menu_root" name="Real Estate">
+        <menuitem id="estate_advertisement_menu" name="Advertisements">
+            <menuitem id="estate_properties_menu_action" action="estate_property_action"/>
```

Comment:
```suggestion
            <menuitem id="action_estate_properties_menu" action="action_estate_property"/>
```
Action name must always start with `action_` followed by what it performs. Please make the corrections in all defined actions